lnbits/db.py

```diff
@@ -126,6 +120,24 @@ def timestamp_placeholder(cls):
         else:
             return "?"
 
+    @classmethod
+    def truncate_date(cls, col: str, trunc: DateTrunc) -> str:
```

Comment:
- value format/truncate should take place at the Business Logic layer (python) and not at the Persistence layer (SQL)

Reply:
Agreed. There are three options to truncate:

- in the db
- in the endpoint
- in the frontend

only the fronend needs it (for this specific use case of charting the payments), so it should be handled there.